OJP/OJP_Trips.xsd

```diff
@@ -1382,7 +1382,7 @@ Can only be used when both participants recognise the same schedule version. If
 			<xs:group ref="BaseTripPolicyGroup"/>
 			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" default="anyPoint">
 				<xs:annotation>
-					<xs:documentation>If a solution for any one of multiple origin/destination points is sufficient. Or a distinct solution for each of the origin/destination points has to be found.</xs:documentation>
+					<xs:documentation>Defines how the router should handle the multiple origins and destinations: If a solution for any one of multiple origin/destination points is sufficient. Or a distinct solution for each of the origin/destination points has to be found. As it is important for the strategy of the distributing trip planning the MultiPointType should be set. If the type is not supported a TRIP_MULTIPOINT_TYPE_NOT_SUPPORTED warning or error must be returned.</xs:documentation>
```

Comment:
I added the remark about the default value because that information would otherwise not appear in the documentation and I think readers might still have the same confusion as revealed by the Austrian comment. 

A second point: I myself don't understand the point of keeping *OJPMultiPointTripRequest.Params.MultiPointType* required while *OJPMultiPointTripRequest.Params* is optional. Wouldn't having *MultiPointType* as optional, together with the default *anyPoint* result in *OJPMultiPointTripRequest.Params.MultiPointType*==*anyPoint* in the case if no *OJPMultiPointTripRequest.Params* is provided, as wanted? And wouldn't that avoid the confusion about the change of optional to required in the hierarchy?